addons/payment_authorize/models/authorize_request.py

```diff
@@ -259,7 +259,7 @@ def _prepare_tx_data(self, token=None, opaque_data=False):
                 }
             }
 
-    def _get_transaction_details(self, transaction_id):
+    def get_transaction_details(self, transaction_id):
```

Comment:
Why did you choose to change this?

Reply:
I using the function outside of the module. I think in that case is better to "make" the method public.